postgrest.cabal

```diff
@@ -1,5 +1,5 @@
 name:               postgrest
-version:            8.0.0.20210921
+version:            8.0.0.20211109
```

Comment:
@monacoremo Don't know if you noticed this, but somehow the `v9.0.0-a1` pre-release was deleted, its assets are gone but the [tag](https://github.com/PostgREST/postgrest/releases/tag/v9.0.0-a1) remains.

Is there some kind of GitHub rule where there can be only one `pre-release`? If so, could this new pre-release delete the previous one?

Reply:
Hm, we are deleting an existing pre-release with the same tag here: https://github.com/PostgREST/postgrest/blob/2b22f889bdd191453c3f2798b5ea9da024231c4f/.github/workflows/ci.yaml#L327 - but this shouldn't  have deleted that one, unless something went wrong?

I updated the version here to invalidate the FreeBSD CI cache which is doing weird things, it shoudn't have triggered anything as there is not tag

Reply:
Here's the log of the last release's CI run:
https://github.com/PostgREST/postgrest/runs/4088373432?check_suite_focus=true

It says:

```
Releasing version v8.0.0.20211102 on GitHub (isprerelease=1)...
release not found
https://github.com/PostgREST/postgrest/releases/tag/v8.0.0.20211102
```

The 2nd line is from the delete command - so it should not have deleted the v9 release.

I noticed the release was gone a few days ago, too - but I assumed somebody deleted it manually because of the accidental version bump?

Reply:
> Is there some kind of GitHub rule [..] where there can be only one pre-release?

That seems unlikely, because we still have the old nightly release around - so we do have 2 pre-releases at the same time right now, in terms of GitHub.

Reply:
>  but I assumed somebody deleted it manually because of the accidental version bump?

Hm, I didn't delete it. Don't know what might have happened.